Add the "pdashv2" URL prefix

portante opened the pull request:
Added the "pdashv2" URL prefix via .webpackrc.js publicPath, updated the
index.html template (src/index.ejs) to reference the pdashv2 prefix for
the favicon.png file, and updated the reference to the config file off of
the current web pages pathname.

k-rister commented:
I would be curious to see if we could implement something along the lines of the following for the template:

`
<html>
  <head>
  ...
  <script src=/static/magic.js></script>
  </head>
  <body>
    <script>
      do_magic();
    </script>
  </body>
</html>
`

Then in `do_magic` (or whatever it's called) we do all the cool stuff like loading the JSON and building the page.  That way the entire thing can be changed later (if required) just be rewriting `magic.js` and the `do_magic` function and the template pages never have to be touched again.

An implementation like this would have made transitioning from v0.2 to v0.3 charts trivial -- as long as we are willing to drop the version from the path and update the scripts in place.

k-rister commented:
FWIW, I find the "insert code" functionality of the comment editor lacking...

portante committed:
Add the "pdashv2" URL prefix

Added the "pdashv2" URL prefix via .webpackrc.js publicPath, updated the
index.html template (src/index.ejs) to reference the pdashv2 prefix for
the favicon.png file, and updated the reference to the config file off of
the current web pages pathname.
